scripts/enterenv

```diff
@@ -1,120 +0,0 @@
-#!/bin/bash
-
-root-init() {
-  mkdir -p /envy/users
-  cp /bin/envcmd /envy/.envcmd
-}
-
-user-init() {
-  if [[ ! -d "/envy/users/$USER" ]]; then
-    mkdir -p "/envy/users/$USER"
-    mkdir -p "/envy/users/$USER/envs"
-    mkdir -p "/envy/users/$USER/home"
-    mkdir -p "/envy/users/$USER/root"
-    cp /tmp/data/home/.bashrc "/envy/users/$USER/home"
-    cp /tmp/data/home/.bashrc "/envy/users/$USER/root"
-  fi
-}
-
-env-init() {
-  if [[ ! -d "/envy/users/$USER/envs/$ENVIRON" ]]; then
-    cp -R /tmp/data/env "/envy/users/$USER/envs/$ENVIRON"
-  fi
-  env-docker
-  cd "/envy/users/$USER/envs/$ENVIRON"
-  if ! docker history "$USER/$ENVIRON" &> /dev/null; then
-    echo "Building environment ... "
-    docker build -t "$USER/$ENVIRON" . &> /dev/null
-  fi
-}
-
-env-docker() {
-  mkdir -p "/envy/users/$USER/envs/$ENVIRON/run"
-  if ! docker diff "$USER.$ENVIRON" &> /dev/null; then
-    docker run -d --privileged \
-      --name "$USER.$ENVIRON" \
-      --hostname "$ENVIRON" \
-      --restart always \
-      --volume "$HOST_ROOT/users/$USER/envs/$ENVIRON/run:/var/run" \
-      progrium/dind > /dev/null
-  fi
-}
-
-env-session() {
-  local session="$USER.$(date +%s)"
-  local status="128"
-  set +e
-  while [[ "$status" == "128" ]]; do
-    session-reload-env "$session"
-    docker rm -f "$session" &> /dev/null
-    docker run -it \
-      --name "$session" \
-      --net "container:$USER.$ENVIRON" \
-      --env "HOSTNAME=$ENVIRON" \
-      --env "ENVY_SESSION=$session" \
-      --env "ENVY_RANDOM=$RANDOM" \
-      --env "ENVY_USER=$USER" \
-      --env "ENV=/env/envyrc" \
-      --env "DOCKER_HOST=unix:///env/run/docker.sock" \
-      --volume "$HOST_ROOT/.envcmd:/sbin/reload:ro" \
-      --volume "$HOST_ROOT/.envcmd:/sbin/rebuild:ro" \
-      --volume "$HOST_ROOT/.envcmd:/sbin/switch:ro" \
-      --volume "$HOST_ROOT/.envcmd:/sbin/commit:ro" \
-      --volume "$HOST_ROOT/users/$USER/envs/$ENVIRON:/env" \
```

Comment:
what happened to the `/env` dir on the go port? do we need to access it from `/envy/users/USER/environs/ENV` from now on? IMHO that seems to be a long dir path to remember :cry: